manifests/0000_70_cluster-network-operator_02_rbac.yaml

```diff
@@ -1,16 +1,16 @@
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: default-account-cluster-network-operator
```

Comment:
You cannot just rename something like this as you'll orphan resources in the cluster, please create a tombstone for the old name so that the CVO removes it from existing clusters on upgrade

Reply:
You need to remove it from the cluster IMO, check out https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/dev/object-deletion.md for explanation of how